SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

```diff
@@ -4,11 +4,31 @@ NOTE:  This file contains the content that profiles BICEPS semantics for use in
 
 // 8.2.8
 [sdpi_offset=8]
-===== SDC/BICEPS semantic content
```

Comment:
@ToddCooper I guess this hierarchy decrement makes sense here, doesn'it it? 8.2.8 is in the comment, it'd missing a fourth digit otherwise.